cypress/e2e/SignInPassword.cy.ts

```diff
@@ -6,15 +8,27 @@ import { MEMBERS } from '../fixtures/members';
 
 describe('Email and Password Validation', () => {
   it('Sign In With Password', () => {
-    const redirectionLink = 'mylink';
+    const redirectionLink = 'http://localhost:3005/mylink';
```

Comment:
This is a url that is different from the baseurl of Auth. I don't think it needs to be in an env var. Basically this could be the address from builder or whatever url we need to redirect too after the login. It is used only here, the second intercept also uses it.

Reply:
But is there an impact if I change my .env.test variables?

Reply:
No there will be no impact, unless you use specifically the 3005 port ...

Reply:
Well, I think even in that case, the intercept should come first, so it would be fine actually.